members/_posts/2017-02-15-dinggesellschaft.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: memberpage
+category: cmember
+logo : gzevd.png
+
+name: 'Ding Gesellschaft'
```

Comment:
We would expect your official legal name here, which would rather be "Gesellschaft zur Entwicklung von Dingen mbH", wouldn't it? Or does this break our layout? ;-)